config.yaml

```diff
@@ -11,10 +11,14 @@ options:
       default: 3
```

Comment:
I can't comment on this directly in this file since it's not being changed, but we may want to use something like `org_or_repo` instead of `path`. When reading the code, I thought `path` was referring to where the binary was stored locally or something, but that's a different variable.